GAE/src/org/waterforpeople/mapping/app/gwt/client/surveyinstance/SurveyInstanceDto.java

```diff
@@ -40,14 +40,23 @@
     private Long surveyalTime = 0L;
 
     private String submitterName;
+    private Double formVersion; //What form version was used to collect the data
     private String deviceIdentifier;
     private String surveyCode;
     private String approvedFlag;
     private Long surveyedLocaleId;
     private String surveyedLocaleIdentifier;
     private String surveyedLocaleDisplayName;
 
-    public String getApprovedFlag() {
+    public Double getFormVersion() {
+		return formVersion;
+	}
+
+	public void setFormVersion(Double formVersion) {
```

Comment:
Formatting is off here

Reply:
Yeah, forgot to set up space-not-tabs when getting new computer...